packages/product-components/package.json

```diff
@@ -17,6 +17,7 @@
         "storybook-build": "storybook build -c .storybook -o .build-storybook"
     },
     "dependencies": {
+        "@suite-common/token-definitions": "workspace:*",
```

Comment:
I haven't look into details, but It looks like adding this dependency is breaking the Connect tests in this PR.
It looks it requires some polyfills or browserify.

Reply:
@enjojoy I think you do not need to use `getContractAddressForNetwork` in your component at all. Then you can remove the lib and it will be ok